testing/main_test.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/magefile/mage/sh"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestSetup tests if Kibana can be run against the current registry
+// and the setup command works as expected.
+func TestSetup(t *testing.T) {
+
+	err := os.Chdir("environments")
+	if err != nil {
+		t.Error(err)
+	}
+
+	// Make sure services are shut down again at the end of the test
+	defer func() {
+		err = sh.Run("docker-compose", "-f", "snapshot.yml", "-f", "local.yml", "down", "-v")
+		if err != nil {
+			t.Error(err)
+		}
+
+	}()
+	// Spin up services
+	go func() {
+		err = sh.Run("docker-compose", "-f", "snapshot.yml", "-f", "local.yml", "up", "--force-recreate")
```

Comment:
How about `docker-compose pull`?

Reply:
Doesn't the pull happening automatically if images are missing?

Reply:
"if images are missing" 

so if you rerun the test locally after a week, you won't get fresh images.

Reply:
Adjusted